### PR TITLE
update reward and add getReward API

### PR DESCRIPTION
**Pull request by AllenIversons**

### CODE REVIEW

**Comment by hanbaofu** [comment]
Update reward and add getReward API

File: evaclient/evaclient.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 	"fmt"
 	"math/big"
 
-	"github.com/Evanesco-Labs/go-evanesco"
+	ethereum "github.com/Evanesco-Labs/go-evanesco"
```
Comment (calehh) [inline evaclient/evaclient.go:27]: remove this change

File: internal/ethapi/api.go [hunk 2]
```diff
@@ -45,6 +44,7 @@ import (
 	"github.com/Evanesco-Labs/go-evanesco/params"
 	"github.com/Evanesco-Labs/go-evanesco/rlp"
 	"github.com/Evanesco-Labs/go-evanesco/rpc"
+	"github.com/davecgh/go-spew/spew"
```
Comment (calehh) [inline internal/ethapi/api.go:47]: reset this change

**Review by calehh — CHANGES_REQUESTED** [review]
reset unrelated changes

**Pull request merged**